pallets/automation-time/rpc/Cargo.toml

```diff
@@ -19,6 +19,7 @@ sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkad
 sp-api = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.23" }
 sp-blockchain = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.23" }
 sp-block-builder = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.23" }
+sp-rpc = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.20" }
```

Comment:
should this be bumped to v0.9.23?

Reply:
oh lordie, thanks so much for the catch haha